### PR TITLE
calendar.md

### PR DESCRIPTION
**Pull request by k1kolev**
Proposal for a first version of a descriptive article of ERP.net Calendar and its capabilities.

### CODE REVIEW

File: features/crm/Calendar.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Calendars are well-known and widespread tools used for event management purposes. 
+ERP.net Calendar empowers **coordinated planning, execution, and tracking** of personal and team events, as well as holding meetings related to decision-making, projects, and other activities.
+
+Its **user-friendly design** offers quick and easy event management and visual orientation throughout the set of appointments.
+
+Virtually all shared calendars can show appointments within the company or group they are shared with.
+ERP.net Calendar, however, is much more than just a typical calendar.
+
+*Most ERP systems* encounter difficulties with centralized event and resource management when a single business has registered more than one legal entity and operates these entities separately from a legal perspective.
```
Comment (i-argentinski) [inline features/crm/Calendar.md:13]: - We do not want to refer to other ERPs. 
- We do not want negative connotation in any aspect.
- We just affirm our advantages.

File: features/crm/Calendar.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+* Proposition
+* Potential Client Inquiry
+* Task
+* Demonstration
+* Message
+* Project Charter
+* Publication
+* Advertising Activity
+* Status Report
+
```
Comment (i-argentinski) [inline features/crm/Calendar.md:57]: What are these? If this is user-defined, this should be specified.

File: features/crm/Calendar.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+Depending on the selected appointment type, ERP.net Calendar **automatically assigns** the new appointment one of the **three system types**:
```
Comment (i-argentinski) [inline features/crm/Calendar.md:58]: This is not automatic. It is the basic type which the user chooses when creating new Activity.

**Review by i-argentinski — CHANGES_REQUESTED** [review]

File: features/crm/Calendar.md (hunk 1 above)
Comment (k1kolev) [reply features/crm/Calendar.md]: Regarding remarks: 

•	We do not want to refer to other ERPs. - Reference to other ERPs removed.
•	We do not want negative connotation in any aspect. - Negative connotation changed.
•	We just affirm our advantages. - Emphasized on multiple legal entities support, user-friendly interface, easy navigation, filtering combinations, and other features.

File: features/crm/Calendar.md (hunk 2 above)
Comment (k1kolev) [reply features/crm/Calendar.md:82]: Regarding remarks: 

What are these? If this is user-defined, this should be specified. - Document types are now described with more clarity. Description of what is system-based and predefined, and what is user-configurable or vendor-configurable.

File: features/crm/Calendar.md (hunk 3 above)
Comment (k1kolev) [reply features/crm/Calendar.md]: Regarding remarks: 

This is not automatic. It is the basic type which the user chooses when creating new Activity. - Fixed the "automatic" statement. Changed to user choice between system types.

File: features/crm/Calendar.md [hunk 4]
```diff
@@ -2,23 +2,34 @@
 
 ## Overview
 
-Calendars are well-known and widespread tools used for event management purposes. 
-ERP.net Calendar empowers **coordinated planning, execution, and tracking** of personal and team events, as well as holding meetings related to decision-making, projects, and other activities.
+Calendars are well-known and widespread tools used for event management purposes. **@@name Calendar** is a valuable tool that empowers **coordinated planning, execution, and tracking** of personal and team assignments, as well as holding meetings related to **decision-making, project management, and other activities**.
```
Comment (i-argentinski) [inline features/crm/Calendar.md:5]: Each sentence should be on new line.

**Review by i-argentinski — CHANGES_REQUESTED** [review]

File: features/crm/Calendar.md (hunk 4 above)
Comment (k1kolev) [reply features/crm/Calendar.md]: Regarding remarks: 

Each sentence should be on new line. - Moved the second sentence on a new line.

**Review by i-argentinski — APPROVED** [review]

**Pull request merged**